packages/axios-extension/src/abap/message.ts

```diff
@@ -120,13 +120,13 @@ function logFullURL({
         const base = new URL(host).origin; // We only care for the origin value
         // Add this instruction to the user because of this bug in VS Code: https://github.com/microsoft/vscode/issues/144898
         // It undoes the encoding of special characters in the URL sent to the browser
-        log.info('Please copy/paste this URL in a browser for more details:');
+        log.debug('Please copy/paste this URL in a browser for more details:');
```

Comment:
Should not be end user messaging in a low level module. The consumer needs to deal with this directly based on a returned message.

Reply:
Makes sense. But better to keep this as info for now until that is addressed?